force-app/main/default/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.js

```diff
@@ -42,6 +42,7 @@ export default class GeBatchGiftEntryHeader extends LightningElement {
             case this.ACTIONS.PROCESS_BATCH:
                 this.dispatchEvent(new CustomEvent('processbatch'));
                 this.isGiftBatchProcessing = true;
+                this._isBatchProcessing = true;
```

Comment:
So i am not sure if this is the best place to place this but it basically allows for the spinner to be triggered immediately after a click event occurs with the Process Batch button

Reply:
I believe adding this boolean here has no effect on the spinner that displays when the batch starts processing, the issue we are trying to solve here is we make an async call to the server and wait for a response before we display a spinner, the spinner is triggered in the `geGiftEntryFormApp`. To fix this, I think we need to add a seperate boolean flag that controls the spinner and will be triggered once the 'Process Batch' button is clicked but will flip its state to `false` based on the result of processing the batch.